client/src/plugins/camunda-plugin/deployment-tool/__tests__/DeploymentToolSpec.js

```diff
@@ -408,7 +408,42 @@ describe('<DeploymentTool>', () => {
     });
 
 
-    it('should not handle deployment error given a non DeploymentError', async () => {
+    it('should not handle deployment error given a non DeploymentError during getVersion', async () => {
+
+      // given
+      const deploymentErrorSpy = sinon.spy(),
+            configuration = createConfiguration(),
+            activeTab = createTab({ name: 'foo.bpmn' });
+
+      const errorThrown = [
+        new ConnectionError({ status: 500 }),
+        new Error()
+      ];
+
+      for (let i = 0; i < errorThrown.length; i++) {
```

Comment:
why not `errorThrown.forEach((errorThrown) => { ... });`

Reply:
To be consistent within the same file (see https://github.com/camunda/camunda-modeler/blob/develop/client/src/plugins/camunda-plugin/deployment-tool/__tests__/DeploymentToolSpec.js#L423)

Reply:
Done